src/etherscan/constants.ts

```diff
@@ -0,0 +1,46 @@
+// source code for NonReceivableInitializedProxySourceCode from https://github.com/PartyDAO/partybid/blob/main/contracts/NonReceivableInitializedProxy.sol
+// todo: should we download this from github rather than hardcode it?
+export const NonReceivableInitializedProxySourceCode = `// SPDX-License-Identifier: MIT
```

Comment:
not a huge deal, but it would feel more natural to me to store this as a `.sol` file in the repo and then read it using JS

Reply:
i'm going to leave this as a js constant for now, and then we should talk about moving this to a `.sol` file, or doing something like downloading it from github or npm.